devices/philips/light_zb3_C.json

```diff
@@ -60,6 +64,8 @@
     "929002376101",
     "929003053301_01",
     "929003053301_02",
+    "929003052501_01",
```

Comment:
Please keep the _Model Identifier_ values sorted.

Reply:
Done, adjusted to ascending sort order.